common/views/components/Header/HeaderSearch.tsx

```diff
@@ -72,7 +72,6 @@ const HeaderSearch = ({
 
   useEffect(() => {
     handleCloseModal();
-    setInputValue('');
```

Comment:
I believe this allowed for the header search bar to be emptied. It wasn't an agreed upon behaviour, but since we're changing it, just checking it was a wanted behaviour change?

https://user-images.githubusercontent.com/110461050/233596807-59e22b37-d858-4b6b-a789-dbfb32c57412.mov